dvc/command/get.py

```diff
@@ -3,14 +3,49 @@
 
 from .base import append_doc_link
 from .base import CmdBaseNoRepo
-from dvc.exceptions import DvcException
+from dvc.exceptions import DvcException, NotDvcRepoError
 
 
 logger = logging.getLogger(__name__)
 
 
 class CmdGet(CmdBaseNoRepo):
+    def _show_url(self):
+        from dvc.api import get_url
+
+        try:
+            url = get_url(
+                self.args.path, repo=self.args.url, rev=self.args.rev
+            )
+            logger.info(url)
+        except NotDvcRepoError:
+            # NOTE: there's no way to get direct links to files inside
+            # git repository.
+            logger.exception(
+                "Could not show url for '{}' from '{}'. ".format(
```

Comment:
Feels weird that here we start with a capital letter and down below with a lowercase. Need to stick to one or another.

Reply:
Good catch. Let me see, what do we do usually. :slightly_smiling_face:

Reply:
@skshetry If we raise proper exc from `get_url` as suggested in https://github.com/iterative/dvc/pull/3156#discussion_r367378039 , we will have only one exc processing snippet, so it will be consistent automatically 🙂